### ISSUE TITLE
On windows gradle sync not work (symlinks in buildSrc of plugin)

### ISSUE THREAD

**Issue by ln-12** [icerockdev/moko-resources#147]
I am using Android Studio 2020.3.1 Canary 2. After cloning and importing the project, the following errors occure:

``` 
e: <path>\moko-resources\plugins\resources-generator\build.gradle.kts:6:5: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public inline fun ObjectConfigurationAction.plugin(pluginClass: KClass<out Plugin<*>>): ObjectConfigurationAction defined in org.gradle.kotlin.dsl

e: <path>\moko-resources\plugins\resources-generator\build.gradle.kts:6:12: Unresolved reference: Deps
```

What do I need to change?

**Comment by Alex009** [comment]
hi! which os you use?
now plugin buildSrc symlinked to root buildSrc and it can broke windows...it will be fixed later

**Comment by ln-12** [comment]
Yeah, I am running Windows 10. No big deal, I just wanted to mention it.

**Comment by Alex009** [comment]
will be available in 0.15.1
